dotCMS/src/main/java/com/liferay/portal/ejb/UserPersistence.java

```diff
@@ -231,31 +240,28 @@
 		}
 	}
 
+	@CloseDBIfOpened
 	protected com.liferay.portal.model.User findByPrimaryKey(String userId)
-		throws NoSuchUserException, SystemException {
+			throws NoSuchUserException, SystemException {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.liferay.portal.SystemException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)